timesketch/lib/datastores/elastic.py

```diff
@@ -594,18 +594,26 @@ def count(self, indices):
             indices: List of indices.
 
         Returns:
-            Number of documents.
+            Tuple containing number of documents and size on disk.
         """
         if not indices:
-            return 0
+            return 0, 0
+
         try:
-            result = self.client.count(index=indices)
-        except (NotFoundError, RequestError):
+            es_stats = self.client.indices.stats(
+                index=indices, metric='docs, store')
+        except NotFoundError:
             es_logger.error(
                 'Unable to count indexes (index not found)',
                 exc_info=True)
-            return 0
-        return result.get('count', 0)
+            es_stats = {}
+
+        doc_count_total = es_stats.get(
+            '_all', {}).get('primaries').get('docs', {}).get('count', 0)
```

Comment:
maybe have {} as the default in other places too

.get('primaries', {}).get('docs....

and .get('primaries', {}).get('store', {}).get..... below

Reply:
good catch, fixed.